### ISSUE TITLE
Anchor point and textbox interaction

### ISSUE THREAD

**Issue by ryanhurststrava** [airbnb/lottie-android#2201]
*Lottie is supported and developed on nights and weekends. Issues from [Lottie sponsors](https://github.com/users/gpeal/sponsorship) will be prioritized.*

**Checklist**

1. My animation doesn't use any [unsupported features](http://airbnb.io/lottie/#/supported-features).
        I'm not sure if this counts as anchor point grouping
3. I know what part of my animation doesn't work.
        The only differences between the two layers in my text file are with the anchor point(k values on the "a:" attribute), and text box(sz and ps attributes)
4. I have created a simplified version of my animation 
       [anchor_point.txt](https://github.com/airbnb/lottie-android/files/10170670/anchor_point.txt)
       This has two layers in it that I would expect to line up (about)on top of each other
5. I have attached the AEP file (as a zip file so it can be attached) that only has the part of the animation that doesn't work.
       I can get this, but it will take a bit of effort and learning

**Describe the bug**
When using a text box and an anchor point, the rending doesn't match the behavior of [web](https://lottiefiles.com/tools/json-editor) or iOS

**What version of Lottie did you test this on?**
2ca5196e77e0a42136d11a416d69b3fe201b193b (master 12/4/22)

**What version of Android did you test this on?**
Android 11 pixel 2

**Steps To Reproduce**

1. load the attached [txt](https://github.com/airbnb/lottie-android/files/10170670/anchor_point.txt) file into the sample app

**Screenshots**
expected(from loading my text file into https://lottiefiles.com/tools/json-editor):
![Screen Shot 2022-12-06 at 3 17 43 PM](https://user-images.githubusercontent.com/106116154/206036972-59b6c9c2-e4f7-41f8-ae22-d67e98107042.png)

actual:
![1670365020](https://user-images.githubusercontent.com/106116154/206037116-fdfb7af2-93b7-4807-984b-42269f59224f.png)

**Comment by gpeal** [comment]
@ryanhurststrava I spent a good amount of time trying to come up with the correct robust algorithm for text y position. I got it working better for most cases and got this one closer but it's not perfect. I'm closing this since I likely won't have the time to revisit it and most paragraph and point text seems to be rendering correctly.

If you want to further refine this, it'll likely be up to you or another external contributor to make more progress.
